packages/address-book/address-book/optimism/tokens/tokens.ts

```diff
@@ -398,6 +398,16 @@ const _tokens = {
       'As a Rocket Pool staker, your role is to deposit ETH into the deposit pool which will enable a node operator to create a new Beacon Chain validator. You can stake as little as 0.01 ETH. In doing so, you will be given a token called rETH. rETH represents both how much ETH you deposited, and when you deposited it.',
     logoURI: '',
   },
+  USDP: {
+    name: 'USD+ Stablecoin',
+    symbol: 'USDP',
+    address: '0x73cb180bf0521828d8849bc8CF2B920918e23032',
+    chainId: 10,
+    decimals: 6,
+    website: 'https://overnight.fi/',
+    description: 'USD+ is USDC that pays you yield daily via rebase.',
```

Comment:
Same with this one. USD+ actually uses yield strategies to earn the yield. It would be helpful to talk more about what it is in the description.

Reply:
```suggestion
    description: 'USD+ is USDC that pays you yield daily via rebase.  It is 100% collateralized with assets immediately convertible into USDC.  Yield is generated via strategies such as lending and stable-to-stable pools. Initial strategies include Aave, Rubicon, and Pika.',
```